### PR TITLE
For #126: added public static method pattern

### PR DESCRIPTION
**Pull request by paulodamaso**
For #126: added public static method pattern and tests

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#162](https://github.com/yegor256/aibolit/issues/162) is assigned to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/206b1868-e870-415c-836e-6ec5acfb0cff); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @acheshkov[/z](https://www.0crat.com/u/acheshkov) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: aibolit/patterns/public_static_method/public_static_method.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        :param filename:
+        :return: Tree
+        """
+        with open(filename, encoding='utf-8') as file:
+            tree = javalang.parse.parse(file.read())
+        return tree
+
+    def value(self, filename: str):
+        tree = self.__file_to_ast(filename).filter(javalang.tree.MethodDeclaration)
+        return [node for path, node in tree if 'public' in node.modifiers and 'static' in node.modifiers]
```
Comment (fanifieiev) [inline aibolit/patterns/public_static_method/public_static_method.py:21]: @paulodamaso You can actually do a little better:
`return [node for _, node in tree if all(elem in node.modifiers for elem in ['public', 'static'])]`

**Comment by paulodamaso** [comment]
@fanifieiev Done, please take a look

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job audit: performer @fanifieiev[/z](https://www.0crat.com/u/fanifieiev)[/z](https://www.0crat.com/u/fanifieiev) didn't make CR comments

**Pull request merged**